README.md

```diff
@@ -1,30 +1,5 @@
 <a name="readme-top"></a>
 
```

Comment:
Great job @danielochuba 🤝

- [ ] As stated in the project requirements, make sure your README contains an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).